packages/web/src/components/usdc-manual-transfer/USDCManualTransfer.tsx

```diff
@@ -116,28 +124,24 @@ export const USDCManualTransfer = ({
           />
         </Flex>
       </Flex>
-      <AddressTile address={USDCUserBank ?? ''} iconLeft={IconLogoCircleUSDC} />
-      <div
-        className={cn(styles.buttonContainer, {
-          [styles.mobile]: isMobile
-        })}
-      >
+      <AddressTile address={USDCUserBank ?? ''} />
```

Comment:
better to just do a `{USDCUserBank ? <AddressTile address={USDCUserBank} /> : null`?